runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
 
 
 
+
+=================================================================
+=                 3.5 SELECTING TO A CHARACTER                  =
+=================================================================
+
+ Type f<ch> to select up to and including (find) a character.
+ Type t<ch> to do the same, but not including ('til) a character.
+ Type uppercase F/T to do the same backwards.
+
+ 1. Move the cursor to the line below marked -->. Place the
+    cursor on the first dash.
+ 2. Type f[ to select to the square bracket.
+ 3. Press d to delete your selection.
+ 4. Go to the end of the line and repeat with F].
+ 5. Move to the second line marked -->, just after the arrow.
+ 6. Use t and T to delete the dashes around the sentence.
+
+ --> -----[Free this sentence of it's brackets!]-----
+ --> ------Free this sentence of it's dashes!------
+
+ Note: Unlike Vim, Helix doesn't limit these commands to the 
+       current line. It searches for the character in the file.
+
+=================================================================
+=                        CHAPTER 3 RECAP                        =
+=================================================================
+
+ * Type C to copy the selection on the line below and Alt-C for
+   above.
+
+ * Press s to select all instances of a regex pattern inside
+   the current selection.
+
+ * Press semicolon ( ; ) to collapse selection.
+
+ * Type f / F to extend selection up to & including a character.
+
+ * Type t / T to extend selection until a character.
+
+
+
+
+
+
+
+=================================================================
+=                 4.1 COPYING AND PASTING TEXT                  =
+=================================================================
+
+ Press y to yank (copy) the selection.
+ Press p to paste the yanked selection after the cursor.
+ Type P to paste the yanked text before the cursor.
+
+ 1. Move the cursor to the line below marked -->.
+    Make sure your cursor is on the "b" of banana.
+ 2. Press w to select "banana" and y to yank it.
+ 3. Move to the space between "2" and "3" and press p to paste.
+ 4. Repeat between "3" and "4".
+
+ --> 1 banana 2 3 4
+     1 banana 2 banana 3 banana 4
+
+ Note: Whenever you delete or change text, Helix will copy the
+       altered text. Use alt-d/c instead to avoid this.
+ Note: Helix doesn't share the system clipboard by default. Type
+       space-y/p to yank/paste on your computer's main clipboard.
+
+=================================================================
+=                     4.2 CHANGING CASE                         =
+=================================================================
+
+ Type ~ to switch the case of all selected letters.
+ Press ` to set all selected letters to lowercase.
```

Comment:
How come one is type and one is press? Are you assuming that based on the keyboard layouts that require shift? On dvorak for programmers this may be quite different from what you think.

Reply:
I guess I'll just change the symbols to "press" I'll need to come up with some sort of ruleset to differentiate when you should use "type"/"press". It seems silly but sometimes it just makes sense to use one or the other so you need a convention to stick to.